### ISSUE TITLE
Time for semi-official fork?

### ISSUE THREAD

**Issue by 0x2206** [dillidon/Alerts-Pickers#69]
It looks this awesome project is not maintained anymore and as world moves on I guess it would be beneficial to all of us to deal with opened issues and pull requests. Last thing I want to do is to create another fork, take other's work and credit it to myself. That is why I propose to focus on one of already created forks and treat it as a successor. Alternatively, @dillidon could designate a new maintainer of this repository.

I am mentioning you, former contributors, as you are probably the best candidates:

@JoeSzymanski
@ss18
@loicgriffie
@jason-ingenuity
@basvankuijck
@kobeumut
@jahid-hasan-polash
@daemedeor
@dongdongpc
@theyongdai

What do you think?

**Comment by ss18** [comment]
I agree with @0x2206, would be nice if @dillidon could set some maintainers.

**Comment by loicgriffie** [comment]
When looking at the repository historic, small chances that @dillidon add maintainers or even take into account this. I agree with a fork to continue support this library with multiple maintainers to avoid the same

**Comment by 0x2206** [comment]
Excellent! Third option is to create organisation account on GitHub and cover fork under it. Let's wait couple of days more for other contributors input and then we move on with the fork.

**Comment by loicgriffie** [comment]
--------------------------------------------------------------------------------
 🎉  Congrats

 🚀  RLBAlertsPickers (1.1) successfully published
 📅  November 9th, 16:31
 🌎  https://cocoapods.org/pods/RLBAlertsPickers
 👍  Tell your friends!
--------------------------------------------------------------------------------
+ Swift 4.2
+ Fix warnings + deprecated
+ TelegramPicker (thx @chingiz888)

@0x2206 @ss18 @chingiz888

**Comment by realmtai** [comment]
Preferably, keeping this project away from cocoapods, it contains too much pixie dust. IMHO, the official/semi-official fork should capture the essence of the original project. 

It's just another XCode project. Anyone can feel free to fork and add any fancy they want, cocoapods, swift package manager or `<insert_fancy_manager_2_0>`.

**Comment by dillidon** [comment]
Hi guys! The birth of a child did not allow me to further develop the project! There are difficulties of its development in using the 'UIAlertController'', instead of which I would like to use a simplified version of [SwiftEntryKit](https://github.com/huri000/SwiftEntryKit), that I use now in my current project.
